mobile-browser-based-version/mobile-app/README.md

```diff
@@ -0,0 +1,66 @@
+
+### Compiles and minifies for production
+```
+npm run build
+```
+
+### Lints and fixes files
+```
+npm run lint
+```
+
+### Customize configuration
+See [Configuration Reference](https://cli.vuejs.org/config/).
+
+## Explenation on the current architecture of the app
+### Use of Vue.js
+The main framework used by the application is Vue.js. It's a widely used framework to build single-page UI using so-called `routers` (See [Reference](https://router.vuejs.org/guide/)).    
+The application is built around Vue.js components. Essentially, components are defined around two parts: 
+1. An HTML template that states how the component should be rendered 
+2. A script that defines the behaviors of the components
+
+### Components architecture of the project 
+Components can be organized in a parent/child fashion. Meaning that we can have one parent component that holds many other child components.  
+`routers` are used to define which components are displayed to the user depending on the user's inputs. 
+
+The application runs the following architecture: 
+- **The global component** of the app is called `App.vue`. This component implements a mini-side bar that is always displayed to the user. This mini-sidebar allows the user to directly access the proposed task list and change some parameters of the page (color and night mode).
+- **Information Display Components** are components that are displayed on the right side of the mini-side bar. Depending on the user's path choice, a component is displayed. The following components can be displayed:
+ - **The task list component** is called `TaskList.vue`. It's the default component used to fill this space. It shows which tasks are open for collaboration. 
+ - **Task-related Components** are components used to display the interface associated with tasks. Because tasks can require many information pages such as task description, or task training information, a task UI requires many components. Again, these components come in a parent-child relation: one global component (called `[taskName]_model.vue`) is used to implement a sidebar that allows the user to navigate through the different components associated with a task. On the right side of this global component, the following components are used to create a task (and note that all of them need to be created for each task): 
+    - **Description of the task** under the name `[taskName]_description.vue. It gives an overview of the task. 
+    - **Training of the task** under the name `[taskName]_training.vue. Allows the users to train the model. As a side note, components are created only when they are called by the user. Meaning that until the user reaches the training page of the task, the `[taskName]_training.vue`is not created. When the user reaches for the first time the page, the component is created, and only then the NN model is created and stored in the browser's indexdb database. 
```

Comment:
clarify that the algorithmic parts are here in this file (for now). the training is mainly generic for all tasks, but the data pre-processing is very task-specific (also accuracy display)